ui/raidboss/data/06-ew/dungeon/aloalo_island.ts

```diff
@@ -51,13 +52,11 @@ const triggerSet: TriggerSet<Data> = {
           return output.first!();
         if (data.quaArmamentsCount === 2)
           return output.second!();
-        if (data.quaArmamentsCount === 3)
-          return output.third!();
-        return output.text!();
```

Comment:
I think the logic here was wrong. The third (and every one after) are all the donuts+axes combination.